event.go

```diff
@@ -685,6 +687,13 @@ func (e *Event) Interface(key string, i interface{}) *Event {
 	return e
 }
 
+// CallerSkipFrame instructs any future Caller calls to skip the specified number of frames.
+// This includes those added via hooks from the context.
+func (e *Event) CallerSkipFrame(skip int) *Event {
+	e.skipFrame++
```

Comment:
Should be `e.skipFrame += skip`

Reply:
bdfa5fe2015622e12817a668b461f15fddfa3f9a